blue/session01/s01_facilitator/mtl_session01_say.md

```diff
@@ -16,153 +16,157 @@ output:
 [<img src = "https://github.com/lzim/teampsd/blob/master/resources/title_slides/mtl_s01_teamvision_title.png"
      height = "175" width = "420">](#DontLink)  
 
-## Today we're modeling to learn how to align our team vision.
```

Comment:
I'm assuming the period is removed because of a rule in the linter, probably something to do with headers and reasoning that headers shouldn't being complete sentences? Is there any way we could make exceptions to this rule or remove it (not sure what other linter errors we may miss or stylistic changes would occur if this were removed though)?

Justification: I think having headers like these as complete sentences make the reading flow smoother for both see and say guides and I think the goal was to point out what is the main topic/focus today by making the sentence big and bold like a header.

Reply:
@lijenn I am with you on this! @jamesmrollins Is it feasible to modify the linter rules to leave periods in places like this?

Reply:
Hi @lijenn and @branscombj  below is the rule that was catching the heading punctuations.  Yes, it is possible to delete this rule entirely or modify it to allow certain punctuations.  The current configuration excepts exclamation points and question marks.

------------------------------------------------
MD026 - Trailing punctuation in heading
Tags: headings, headers

Aliases: no-trailing-punctuation

Parameters: punctuation (string; default ".,;:!。，；：！")

Fixable: Most violations can be fixed by tooling

This rule is triggered on any heading that has one of the specified normal or full-width punctuation characters as the last character in the line:

# This is a heading.
To fix this, remove the trailing punctuation:

# This is a heading
Note: The punctuation parameter can be used to specify what characters count as punctuation at the end of a heading. For example, you can change it to ".,;:" to allow headings that end with an exclamation point. Question mark is allowed by default because of how common it is in headings of FAQ-style documents. Setting the punctuation parameter to "" allows all characters - and is equivalent to disabling the rule.

Note: The trailing semicolon of HTML entity references like &copy;, &#169;, and &#x000A9; is ignored by this rule.

Rationale: Headings are not meant to be full sentences. More information: https://cirosantilli.com/markdown-style-guide#punctuation-at-the-end-of-headers

-------------------------------------------------------------------

## See Boldface Below For Style Rationale

From [Ciro Santilli](https://cirosantilli.com/markdown-style-guide/#punctuation-at-the-end-of-headers)

Punctuation at the end of headers

Don’t add a trailing colon : to headers.

**Rationale: every header is an introduction to what is about to come next, which is exactly the function of the colon.**

Don’t add a trailing period . to headers.

**Rationale: every header consists of a single short sentence, so there is not need to add a sentence separator to it.**

Good:

# How to do make omelet

Bad:

# How to do make omelet:

Bad:

# How to do make omelet.

---------------------------------------------

_Finally, Someone Would Have to Go Back And Put All Those Periods and Colons Back In_